dvc/repo/artifacts.py

```diff
@@ -158,10 +158,31 @@ def get_path(self, name: str):
         dirname = m.group("dirname")
         if dirname:
             dirname = dirname.rstrip(SEPARATOR_IN_NAME)
+
+        return dirname, m.group("name")
+
+    def get_path(self, name: str):
```

Comment:
This also changes the expectation of `name` in that, it is always expected to contain a `dirname` component relative to the root of the git repo.